components/01-atoms/forms/textfields/_yds-textfields.scss

```diff
@@ -14,6 +14,14 @@ $form-item-icon-size: 1.7rem;
   margin-bottom: var(--size-spacing-7);
   max-width: $form-item-max-width;
   clear: both;
+
+  .form--inline & {
+    margin-bottom: unset;
+  }
+}
+
+.search-form--page.form--inline {
+  margin-bottom: var(--size-spacing-8);
 }
```

Comment:
@ShaneMcPartlinYale This is super minor, but I would remove this code block
```
.search-form--page.form--inline {
  margin-bottom: var(--size-spacing-8);
}
```

And apply the margin-bottom in the search-result file here: `/component-library-twig/components/02-molecules/search-result/_yds-search-result.scss`(

And simply add `margin-bottom: var(--size-spacing-8);` on line 13, after the `gap` property, like this: 
```
.search-form--page {
  display: flex;
  gap: var(--size-spacing-5);
  margin-bottom: var(--size-spacing-8);
}
```

Reply:
I have made these changes in the latest commit. Thanks for pointing it out!